docs/modules/ROOT/pages/security-guide/ws-securitypolicy-auth.adoc

```diff
@@ -0,0 +1,120 @@
+[[ws-securitypolicy-auth]]
```

Comment:
@rhaetor this is the simplyfied docs page with references to QCXF test code that we spoke about. Would you like to review, before I merge?